agent: Add commit ID to eval telemetry

tmickleydoyle opened the pull request:
Release Notes:

- N/A

maxdeviant committed:
proto: Add `ZedProTrial` to `Plan` (#28885)

This PR adds the `ZedProTrial` member to the `Plan` enum.

Release Notes:

- N/A

tmickleydoyle committed:
Add commit id to eval telemetry

tmickleydoyle committed:
Update clippy error

maxdeviant commented on crates/eval/src/eval.rs:
```diff
@@ -271,12 +271,13 @@ fn main() {
                                 Ok(judge_output) => {
                                     const SCORES: [&str; 6] = ["💀", "😭", "😔", "😐", "🙂", "🤩"];
+                                    // Ensure score is u32 and properly convert to usize for indexing
+                                    let score_u32: u32 = judge_output.score;
```
We don't need to have the suffix with the type here:

```suggestion
                                    let score: u32 = judge_output.score;
```

mikayla-maki committed:
Remove bottom dock layout button (#28876)

Release Notes:

- Preview: Removed the layout button from the title bar. The
`bottom_dock_layout` setting still functions.
- Added a setting, `bottom_dock_layout`, for controlling the
relationship between the bottom dock and the left and right docks.

tmickleydoyle committed:
Update crates/eval/src/eval.rs

Co-authored-by: Marshall Bowers <git@maxdeviant.com>

bennetbo committed:
agent: Allow quoting selection when text thread is active (#28887)

This makes the `assistant: Quote selection` work again for text threads.
Next up is supporting this also in normal threads.

Release Notes:

- agent: Add support for inserting selections (assistant: Quote
selection) into text threads

ConradIrwin committed:
Fix anchor_in_excerpt on replaced excerpts (#28880)

Release Notes:

- N/A

maxdeviant committed:
collab: Add `plan` column to `subscription_usages` (#28889)

This PR adds a `plan` column to the `subscription_usages` table.

These tables don't have any records in them yet, so it's fine to make
the column required without a default.

Release Notes:

- N/A

tmickleydoyle committed:
rebase
